src/engine/BMGameAction.php

```diff
@@ -687,24 +687,68 @@ protected function message_size_change(array $preInfo, array $postInfo) {
     /**
      * Describes a die recipe change
      *
+     * There are six possible scenarios here:
+     * - no recipe change
+     * - a non-mood recipe change
+     * - a mood recipe no change
+     * - a mood recipe change
+     * - a non-mood recipe change with a mood recipe no change
+     * - a non-mood recipe change with a mood recipe change
```

Comment:
Nit: why not put examples for each of these, e.g.:

```
* - a mood recipe no change: `(X=6)? => (X=6)?`
```

Reading this initially, i'm not sure what the last two items mean --- i'm sure it's obvious and my brain's just not online yet, but an example would make it obvious regardless.